ionic2/app/pages/map/map.page.html

```diff
@@ -10,4 +10,7 @@
   </button>
 
   <poke-poi-card #poiCard></poke-poi-card>
+
+  <button id="artificial-mob-button"
```

Comment:
Just make sure neither mock data nor the button remain after all.
I like being able to test it that way though.